site/content/docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+Leo urna molestie at elementum eu facilisis sed odio. Non nisi est sit amet facilisis.
```

Comment:
Just curiosity, we have several files with "Lorem ipsum" content (this, doc section, posts section, 0.30.0 section, 0.40.0 section, plugins sections). Do you want to include it to keep having the structure for the future? Is it possible to remove them? WDYT?